cleanlab_studio/internal/tlm/validation.py

```diff
@@ -235,11 +234,15 @@ def process_response_and_kwargs(
 
                         if v is not None and not 0 <= v <= 1:
                             raise ValidationError("Perplexity values must be between 0 and 1")
-
                 else:
                     raise ValidationError(
                         f"Invalid type {type(val)}, perplexity must be either a sequence or a float"
                     )
+            elif key == "explanation":
+                if not isinstance(response, bool):
+                    raise ValidationError(
+                        "Explanation value must be a boolean. Set it to True if you want explanation regarding the score returned."
+                    )
```

Comment:
I'm confused about this error message.

Can you update your PR description with the intended way to activate explainability?

My understanding was:
```
options = {..., "log": ["explanation"]}
tlm = studio.TLM(options=options)
tlm.prompt(...)  # will always return str explanations
tlm.get_trustworthiness_score(...)  # will always return str explanations
```

so I don't understand why explanation would need to have a boolean value.

Reply:
AFAIK there is also no way to pass explanations back into any TLM method

Reply:
This chunk of code is in the wrong place (and what Jonas mentioned is right, I don't think we should be expecting a boolean either). 

This helper function processes the kwargs for `tlm.get_trustworthiness_score` (not what gets passed into the `log` options). See here instead: https://github.com/cleanlab/cleanlab-studio/blob/main/cleanlab_studio/internal/tlm/validation.py#L185-L194

Reply:
oh good catch, that was a legacy way of passing explanation. Above should be correct now!